app/utils/callback/lava.py

```diff
@@ -486,4 +487,6 @@ def add_tests(job_data, lab_name, db_options, base_path=utils.BASE_PATH):
             if errors:
                 raise utils.errors.BackendError(errors)
 
+        store_lava_json(job_data, suite_data)
```

Comment:
It seems to me that if you put this in the loop that iterates over all the test suites, the same JSON file will be created each time but in a different directory with the suite name.  Should this not be called only once outside the loop, like for boots?

Also, keep in mind that `add_test` is called for every callback received, so boots will have their JSON data stored twice with this approach.  As I don't think this was ever actually used for boot data in the past, being a new thing with tests it could be only called in `add_tests` and not in `add_boot`.